core/src/main/scala/org/apache/spark/storage/BlockManagerDecommissioner.scala

```diff
@@ -125,7 +127,11 @@ private[storage] class BlockManagerDecommissioner(
                   logDebug(s"Migrated sub-block $blockId")
                 }
               }
-              logInfo(s"Migrated $shuffleBlockInfo to $peer")
+              val endTime = System.nanoTime()
+              val duration = Duration(endTime - startTime, NANOSECONDS)
+              val totalBlockSize = Utils.bytesToString(blocks.map(b => b._2.size()).sum)
```

Comment:
hmm... if `duration` and `totalBlockSize` are defined as `val`, they will be calculated even if `log.isInfoEnabled` is `false`

Reply:
True, it's probably not a lot of compute though. We could use a lazy val here though.